distributed/bokeh/components.py

```diff
@@ -0,0 +1,256 @@
+from __future__ import print_function, division, absolute_import
+
+from bisect import bisect
+
+from bokeh.models import (
+    ColumnDataSource, Plot, Datetime, DataRange1d, Rect, LinearAxis,
+    DatetimeAxis, Grid, BasicTicker, HoverTool, BoxZoomTool, ResetTool,
+    PanTool, WheelZoomTool, Title, Range1d, Quad, Text, value
+)
+
+from distributed.diagnostics.progress_stream import progress_quads, nbytes_bar
```

Comment:
should migrate these methods as "parse_message" methods on relevant DashboardComponent class.